lib/okta/password-grant.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var xtend = require('xtend');
+
+var requestExecutor = require('./request-executor');
+
+function passwordGrant(config, userFormSubmission, callback) {
+  var req = {
+    url: config.okta.org + 'oauth2/' + config.okta.authorizationServerId + '/v1/token',
```

Comment:
Although if we change the example to not show a trailing '/', then this would need to change. I'm okay with either way, but think we should be consistent (or when processing the config, normalize it).